grpc/src/main/java/com/linecorp/armeria/server/grpc/AbstractServerCall.java

```diff
@@ -540,7 +540,14 @@ static HttpHeaders statusToTrailers(
         GrpcTrailersUtil.addStatusMessageToTrailers(
                 trailersBuilder, status.getCode().value(), status.getDescription());
 
-        MetadataUtil.fillHeaders(metadata, trailersBuilder);
+        try {
+            MetadataUtil.fillHeaders(metadata, trailersBuilder);
+        } catch (Exception e) {
+            // Catching exception is necessary, because server implementer could have set corrupted metadata.
+            return trailersBuilder
+                    .set(GrpcHeaderNames.GRPC_STATUS, "13")
```

Comment:
Can we extract this into a `private static final` field?

```
private static final String INTERNAL_ERROR_CODE_STR =
    String.valueOf(Status.Code.INTERNAL_ERROR.value());
```

Reply:
Yep.
nits;

How about rewording to

```
    private static final String GRPC_STATUS_CODE_INTERNAL =
            String.valueOf(Status.Code.INTERNAL.value());
```

Code.INTERNAL is 13.
Java is typed language, so I thinkg `_STR` is extra one.